app/src/main/java/org/fossasia/openevent/app/core/session/list/SessionsFragment.java

```diff
@@ -77,9 +82,19 @@ public void onStart() {
         getPresenter().attach(trackId, this);
         getPresenter().start();
 
+        binding.createSessionFab.setOnClickListener(view -> openCreateSessionFragment());
+
         initialized = true;
     }
 
+    public void openCreateSessionFragment() {
+        getFragmentManager().popBackStack();
```

Comment:
Why popping first? Please document this

Reply:
I'm doing this so that on pressing back in `CreateSessionFrgament`, the user goes back to the Dashboard.